src/rules/__tests__/label-has-for.test.js

```diff
@@ -21,8 +25,8 @@ makeRuleTester("label-has-for", rule, {
     "<label for='id'><slot /></label>",
     "<label for='id'><div /></label>",
```

Comment:
Wouldn't it be nice if we added a negative test for the custom input components?